support/util/util.go

```diff
@@ -245,3 +245,18 @@ func ConvertImageRegistryOverrideStringToMap(envVar string) map[string][]string
 
 	return imageRegistryOverrides
 }
+
+// IsIPv4 function parse the CIDR and get the IPNet struct if the IPNet.IP cannot be converted to 4bytes format,
+// the function returns nil, if it's an IPv6 it will return nil.
+func IsIPv4(cidr string) (bool, error) {
+	_, ipnet, err := net.ParseCIDR(cidr)
+	if err != nil {
+		return false, fmt.Errorf("error validating the ClusterNetworkCIDR from HostedCluster: %v", err)
```

Comment:
this function is agnostic from HostedCluster, it just receives an string through the signature. So the error should not mention a HostedCluster.